src/pages/Main/Components/Product/components/ProductDetail/ProductDetail.jsx

```diff
@@ -1,92 +1,122 @@
+  };
+  console.log(cart);
   return (
-    <>
-      <div className="productDetailPageTop">
-        <div className="productDetailLeft">
-          <img src={state.img} alt="" />
-        </div>
-        <div className="productDetailRight">
-          <span className="productDetailCategory">{state.type}</span>
-          <h1 className="productDetailName">{state.name}</h1>
-          <p className="productDetailContent">화이팅</p>
-          <div className="productDetailOthers">
-            <div>
-              <ImLink />
-              <BsFacebook />
-              <BsHeart />
-            </div>
-            <div className="productDetailPrice">
-              <span>{state.price}</span>원
-            </div>
+    product && (
+      <>
+        <div className="productDetailPageTop">
+          <div className="productDetailLeft">
+            <img src={product.img} alt="" />
           </div>
-          <select
-            name="product"
-            className="productDetailSelect"
-            defaultValue={''}
-          >
-            <option value="default" selected>
-              추가상품 선택
-            </option>
-            <option value="1">상품1</option>
-            <option value="2">상품2</option>
-          </select>
-          <div className="productDetailPurchase">
-            <div className="productDetailQuatity">
-              <p>구매수량</p>
-              <div className="productDetailCount">
-                <FiMinus onClick={handleCountDown} />
-                <p>{count}</p>
-                <FiPlus onClick={handleCountUp} />
+          <div className="productDetailRight">
+            <span className="productDetailCategory">{product.type}</span>
+            <h1 className="productDetailName">{product.name}</h1>
+            <p className="productDetailContent">화이팅</p>
+            <div className="productDetailOthers">
+              <div>
+                <ImLink />
+                <BsFacebook />
+                <BsHeart />
+              </div>
+              <div className="productDetailPrice">
+                <span>{product.price}</span>원
               </div>
             </div>
-            <p className="productDetaileDelivery">50000만원이상 시 무료배송</p>
-          </div>
-          <div className="productDetailSum">
-            <p className="productDetailSumText">상품금액 합계</p>
-            <p className="productDetailSumTotalPrice">
-              <span>{totalPrice}</span>원
-            </p>
-          </div>
-          <div className="productDetailButtons">
-            <button className="productDetailGift">선물하기</button>
-            <button className="productDetailCart">장바구니</button>
-            <button className="productDetailBuy">바로구매</button>
+
+            <div className="productDetailPurchase">
+              <div className="productDetailQuatity">
+                <p>구매수량</p>
+                <div className="productDetailCount">
+                  <FiMinus onClick={() => handleQuantity('minus')} />
```

Comment:
```suggestion
                  <FiMinus onClick={handleQuantity} />
```
- type이 minus에 대한 처리는 없어서 그냥 이렇게 호출해도 되겠네요